alphagov/paas-cf
# [#127165717] Revert "Update retention period of existing instances"

## What

This reverts commit 788c32d40937459001c99dbd10457881b90f3d5a.

This task and script were used to modify existing Postgres service instances
to enable backups. Any service instances created since then will have this
configuration set by the broker. So we no longer need to run this script in
the pipeline.

## How to review

1. Deploy this branch to your dev Deployer Concourse.
1. Run the `create-bosh-and-cloudfoundry` pipeline up to and including `post-deploy`.
1. Confirm that all of the jobs pass.
1. Confirm that the `post-deploy` job hasn't run a `update-rds-broker-backup-retention-period
` task.

## Who can review

Not @dcarley
